pixeltable/share/packager.py

```diff
@@ -69,138 +70,122 @@ def __init__(self, table: catalog.Table, additional_md: Optional[dict[str, Any]]
 
     def package(self) -> Path:
         """
-        Export the table to a tarball containing Iceberg tables and media files.
+        Export the table to a tarball containing Parquet tables and media files.
         """
         assert not self.tmp_dir.exists()  # Packaging can only be done once per TablePackager instance
         _logger.info(f"Packaging table '{self.table._path}' and its ancestors in: {self.tmp_dir}")
         self.tmp_dir.mkdir()
         with open(self.tmp_dir / 'metadata.json', 'w', encoding='utf8') as fp:
             json.dump(self.md, fp)
-        self.iceberg_catalog = sqlite_catalog(self.tmp_dir / 'warehouse')
+        self.tables_dir = self.tmp_dir / 'tables'
+        self.tables_dir.mkdir()
         with Env.get().begin_xact():
-            ancestors = (self.table, *self.table._base_tables)
-            for t in ancestors:
-                _logger.info(f"Exporting table '{t._path}'.")
-                self.__export_table(t)
+            for tvp in self.table._tbl_version_path.ancestors:
```

Comment:
Why not just call TVP.get_tbl_versions() here? It doesn't look like you're using the path.

Reply:
ok